tests/integration_test.rs

```diff
@@ -1 +1,58 @@
-// todo
+use core::str;
+use std::path::Path;
+
+use assert_cmd::Command;
+use rstest::{fixture, rstest};
+use tempfile::TempDir;
+
+#[fixture]
+fn tmpdir() -> TempDir {
+    TempDir::new().expect("failed to create temp dir")
+}
+
+fn validate_prefix_micromamba(target: &TempDir) {
+    let micromamba_output = Command::new("micromamba")
+        .arg("-p")
+        .arg(target.path())
+        .arg("run")
+        .arg("which")
+        .arg("python")
+        .output()
+        .expect("failed to execute micromamba on prefix");
+
+    let micromamba_stdout = str::from_utf8(&micromamba_output.stdout).unwrap();
+    assert!(micromamba_stdout.trim_end().ends_with("/bin/python"));
+}
+
+fn validate_prefix_conda(target: &TempDir) {
+    let conda_output = Command::new("conda")
+        .arg("run")
+        .arg("-p")
+        .arg(target.path())
+        .arg("which")
+        .arg("python")
+        .output()
+        .expect("failed to execute conda on prefix");
+
+    let conda_stdout = str::from_utf8(&conda_output.stdout).unwrap();
+    assert!(conda_stdout.trim_end().ends_with("/bin/python"));
+}
+
+#[rstest]
+fn test_install(tmpdir: TempDir) {
+    let test_dir = Path::new("tests/test-env");
+    let target = tmpdir;
+
+    Command::cargo_bin("pixi-install-to-prefix")
```

Comment:
i prefer to not use cargo_bin but the proper entrypoint as it makes debugging harder.
(for a single test, it makes sense to use cargo_bin as well, just to see that the whole clap integration works, but for general testing, running the functions directly is better as you see nicer error messages etc)

Reply:
yeah this was meant to be this one single test^^
Currently, we have everything in main, and we can't directly call that here, can we?

Reply:
We can also keep #7 open if you think there is still some work needed (there are also other CLI flags, for example).